engine/src/main/java/org/terasology/engine/audio/events/PlaySoundEvent.java

```diff
@@ -8,20 +8,29 @@
 import org.terasology.engine.network.BroadcastEvent;
 
 /**
- * This event is sent whenever a sound is to be played.
- * The event is not sent to or played for the client of the entity that sent the event as they have simulated it already.
+ * A <i>trigger event</i> to play a sound.
+ * <p>
+ * The event is not sent to or played for the client of the entity that sent the event as they have simulated it
+ * already.
```

Comment:
I'm not sure what you are trying to say here?

```suggestion
 * The event is not sent to or played for the client of the entity that sent the event as they have simulated it
 * already.
```

Reply:
yeah, me neither - I'll have another look at this...